src/pages/junhuihan/Main/Main.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+
+import MainStory from "../components/MainStory/MainStory";
+import Nav from "../components/Nav/Nav";
+import MainAside from "../components/MainAside/MainAside";
+import "./Main.scss";
+
+class Main extends Component {
+  state = {
+    userComments: [],
+  };
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/data.json", {
+      method: "GET",
+    })
```

Comment:
```suggestion
    fetch("http://localhost:3000/data/data.json")
```
- default Method가 "GET"이기 때문에 생략할 수 있습니다.